Project.toml

```diff
@@ -11,8 +11,8 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Quaternions = "0.4"
-StaticArrays = "0.10, 0.11, 0.12, 1.0"
-julia = "0.7, 1"
+StaticArrays = "1.2.12"
```

Comment:
This is related with [this release](https://github.com/JuliaArrays/StaticArrays.jl/releases/tag/v1.2.12)

>Expand signature of `normalize` from `StaticVector` to `StaticArray`